grafana/provisioning/alerting/disk_endurance_low_alert_rule.yaml

```diff
@@ -50,9 +50,9 @@ groups:
         execErrState: Error
         for: 5m
         annotations:
-          description: >-
-            The {{ $labels.disk_type }} at drive bay {{ $labels.drive_bay }} on node
-            {{ $labels.node_id }} has less than 20% disk endurance remaining. Please
-            consider replacing this disk soon.
+          description: >-    
+            Disk {{ $labels.disk_type }} in drive bay {{ $labels.drive_bay }} on node
+            {{ $labels.node_id }} has less than 20% disk endurance remaining. Replace
+            this disk as soon as possible.
```

Comment:
So the tricky thing here is they don't need to replace this disk immediately. They probably want to have a spare on-hand for when they do need to replace it, but it may live for a while (even over a year) past this point.

Reply:
Also a good point. I rewrote this as a more general recommendation.